core/templates/pages/footer_js_libs.html

```diff
@@ -6,12 +6,12 @@
   <% if (webpackConfig.mode == 'production') { %>
     <script src="/build/webpack_bundles/<%= htmlWebpackPlugin.files.js[chunk] %>"></script>
   <% } else { %>
-    <script src="/webpack_bundles/<%= htmlWebpackPlugin.files.js[chunk] %>"></script>
```

Comment:
`webpack` code splitting was not working. Webpack was looking for chunks in the `/` folder.
So, I have added `/webpack_bundles/` public path in the webpack config.